source/MaterialXGenShader/Util.cpp

```diff
@@ -114,40 +114,20 @@ void getFilesInDirectory(const std::string& directory, StringVec& files, const s
 
 bool readFile(const string& filename, string& contents)
 {
-#if defined(_WIN32)
-    // Protection in case someone sets fmode to binary
-    int oldMode;
-    _get_fmode(&oldMode);
-    _set_fmode(_O_TEXT);
-#endif
-
-    bool result = false;
-
-    std::ifstream file(filename, std::ios::in );
+    std::ifstream file(filename, std::ios::in | std::ios::binary);
```

Comment:
Should it say std::ios::binary here? The files given are normally source code files in plain text.

Reply:
As far as I know it just means on Windows skip adding in a line-feed at the end of line. The << operator handles the text processing.

Reply:
ok cool